lib/bulk_data_test_kit/v2.0.0/client/urls.rb

```diff
@@ -0,0 +1,47 @@
+# frozen_string_literal: true
+
+module BulkDataTestKit
+  module BulkDataV200
+    module Client
+      # URLs for use in Bulk Data Client tests and endpoints
+      module URLs
+        RESUME_PASS_PATH = '/resume_pass'
+        BASE_ROUTE = '/:export_id/fhir'
+        METADATA_ROUTE = "#{BASE_ROUTE}/metadata".freeze
+        PATIENT_KICKOFF_ROUTE = "#{BASE_ROUTE}/:type/$export".freeze
+        GROUP_KICKOFF_ROUTE = "#{BASE_ROUTE}/:type/:group_id/$export".freeze
+        SYSTEM_KICKOFF_ROUTE = "#{BASE_ROUTE}/$export".freeze
+        STATUS_ROUTE = '/status/:export_id'
+        OUTPUT_ROUTE = '/output/:export_id.ndjson'
+
+        def base_url
+          "#{Inferno::Application['base_url']}/custom/#{suite_id}"
+        end
+
+        def resume_pass_url
+          base_url + RESUME_PASS_PATH
+        end
+
+        def kickoff_url(export_id)
+          base_url + BASE_ROUTE.gsub(':export_id', export_id)
+        end
+
+        def status_url(export_id)
+          base_url + STATUS_ROUTE.gsub(':export_id', export_id)
+        end
+
+        def output_url(export_id)
+          base_url + OUTPUT_ROUTE.gsub(':export_id', export_id)
+        end
+
+        def suite_id
```

Comment:
Why does the suite_id need to be different for suite_endpoint? The suite id is just used to create the base url, so it should be able to just be `self.class.suite.id`

Reply:
The URLs module is also used in the endpoints (not just suite/tests/groups), which doesn't have `suite` on it's class - so this extra case makes it work there by looking at the result that is available via the test id lookup. I might have missed something in the way I'm using the endpoints though that would make this work? I wasn't able to see an easy way when I was looking at it.

Reply:
Okay yeah I see where you are using the suite_id in the url in the endpoints, so that makes sense. Not sure if there is a better way to make this work, I haven't seen this come up in other test kits. This solution definitely works and looks good to me but if there is a better way of addressing this issue, I am not entirely sure what that would be.